include/benchmark/benchmark.h

```diff
@@ -465,7 +465,7 @@ inline BENCHMARK_ALWAYS_INLINE void DoNotOptimize(Tp const& value) {
 }
 
 template <class Tp>
-inline BENCHMARK_ALWAYS_INLINE void DoNotOptimize(Tp& value) {
+inline BENCHMARK_ALWAYS_INLINE void DoNotOptimize(Tp&& value) {
```

Comment:
i believe this should be inside the defined(BENCHMARK_HAS_CXX11) section below

Reply:
fixed